src/sutta_publisher/publishers/base.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+
+from sutta_publisher.shared.value_objects.results import IngestResult
+
+
+class Publisher(ABC):
+    @abstractmethod
+    def publish(self, result: IngestResult) -> None:
+        pass
+
+
+class ActivePublishers(tuple):
```

Comment:
Yes, we want to go with immutability wherever possible. Here we want to have a list of publishers, but python list is mutable, so instead of reinventing the immutable list, we just use a tuple.